### ISSUE TITLE
to_raster seperate 

### ISSUE THREAD

**Issue by mmann1123** [jgrss/geowombat#52]
Its late and this is a placeholder, but I think I found a problem with to_raster with separate = True (unless Im missing something). 

```
            ds.gw.to_raster('./landuse_stack_2019.tif',
                n_workers=1, n_threads=12,bigtiff=True, compress='lzw',
                separate=True, readxsize=3341, readysize= 2586, out_block_type='gtiff' )
        
2010
100%|███ | 4/4 [00:32<00:00,  8.17s/it]
Traceback (most recent call last):
  File "<stdin>", line 16, in <module>
  File "/home/mmann1123/Documents/github/geowombat/geowombat/core/geoxarray.py", line 719, in to_raster
    **kwargs)
  File "/home/mmann1123/Documents/github/geowombat/geowombat/core/io.py", line 884, in to_raster
    group_keys = list(root.group_keys())
AttributeError: 'NoneType' object has no attribute 'group_keys'
>>> 
```

**Comment by jgrss** [comment]
I think this was just an issue of trying to get `zarr` data even though you specified 'gtiff'. I pushed a fix to `1.3.1`. 

Your data should actually be written because the error occurred in the compression stage. Can you let me know if that was not the case?

**Comment by mmann1123** [comment]
Yeah it was written, thanks for catching that!
